packages/keyboardfocus/src/KeyboardFocusContainer.spec.tsx

```diff
@@ -16,73 +17,48 @@ describe('KeyboardFocusContainer', () => {
   const BasicExample = () => (
     <KeyboardFocusContainer>
       {({ getFocusProps, keyboardFocused }) => (
-        <div {...getFocusProps({ 'data-test-id': 'trigger', 'data-focused': keyboardFocused })}>
-          trigger
-        </div>
+        <div {...getFocusProps({ 'data-focused': keyboardFocused })}>trigger</div>
       )}
     </KeyboardFocusContainer>
   );
 
   describe('getFocusProps', () => {
     describe('onFocus', () => {
       it('should not apply focused prop if focused by mouse', () => {
-        const { container, getByTestId } = render(<BasicExample />);
+        const { container, getByText } = render(<BasicExample />);
 
-        fireEvent.mouseDown(container);
+        userEvent.click(container);
         jest.runOnlyPendingTimers();
-        expect(getByTestId('trigger')).toHaveAttribute('data-focused', 'false');
+        expect(getByText('trigger')).toHaveAttribute('data-focused', 'false');
       });
 
       it('should apply focused prop if focused by keyboard', () => {
-        const { getByTestId } = render(<BasicExample />);
-        const trigger = getByTestId('trigger');
+        const { getByText } = render(<BasicExample />);
+        const trigger = getByText('trigger');
 
-        fireEvent.focus(trigger);
+        userEvent.tab();
         expect(trigger).toHaveAttribute('data-focused', 'true');
       });
 
       it('should apply focused prop if focused by keyboard after mouse event', () => {
-        const { getByTestId } = render(<BasicExample />);
-        const trigger = getByTestId('trigger');
+        const { getByText } = render(<BasicExample />);
+        const trigger = getByText('trigger');
 
-        fireEvent.mouseDown(trigger);
+        userEvent.click(trigger);
         jest.runOnlyPendingTimers();
 
         expect(trigger).toHaveAttribute('data-focused', 'false');
 
-        fireEvent.focus(trigger);
+        userEvent.tab({ shift: true });
+        userEvent.tab();
         expect(trigger).toHaveAttribute('data-focused', 'true');
       });
     });
 
-    describe('onMouseDown', () => {
-      it('should not apply focused prop if mouseddown', () => {
-        const { getByTestId } = render(<BasicExample />);
-        const trigger = getByTestId('trigger');
-
-        fireEvent.mouseDown(trigger);
-        jest.runOnlyPendingTimers();
-
-        expect(trigger).toHaveAttribute('data-focused', 'false');
-      });
-    });
-
-    describe('onPointerDown', () => {
```

Comment:
Tests around `onMouseDown` and `onPointerDown` have been removed because tests using `userEvent.click(trigger)` test these cases.